README.md

```diff
@@ -291,14 +307,18 @@ Deploying task definition.......................................................
 Deployment successful
 ```
 
-Note - if the IMAGE_TAG argument is not provided to the deploy command, the name of the current branch is used as the IMAGE_TAG
+Note - if the IMAGE_TAG argument is not provided to the deploy
+command, the name of the current branch is used as the IMAGE_TAG
 
-You can also check that your environment is functioning after a successful deployment 
-by running the following commands without problems on the master branch:
+You can also check that your environment is functioning after a
```

Comment:
Confusing. Is the implication that my local environment might be broken by the deploy somehow? If not, why is this with the deploy instructions?

Reply:
Deleted.